### DISCUSSION TITLE
docker.stackable.tech dont resolves in Dockerfile

### DISCUSSION THREAD

**Discussion by supsupsap** [stackabletech/spark-k8s-operator#392] [Q&A] [answered]
I want to build Dockerfile with `FROM stackable/image/java-base` and docker cant resolve it.
also tried `FROM docker.stackable.tech/image/java-base`
but got
`"fatal error: docker.stackable.tech/image/java-base: docker.stackable.tech/image/java-base:latest: not found"
`

**Comment by razvan** [comment]
You cannot use `java-base` as starting image because it's an intermediary image used in the SDP  build process. This image is  never published.

If you want to build a custom image that is compatible with the SDP operators you have to start with publicly available product images like: `docker.stackable.tech/stackable/spark-k8s:3.5.1-stackable24.3.0`

**Comment by lfrancke** [comment, marked as answer]
I'm sorry for the terse answer, I'm only on my phone. 

You'll need to follow the instructions from the README. Our base images are not published anywhere, they are only produced as intermediate artifacts during a build.

We use the docker buildx bake functionality to build a kind of "dependency graph" between images.

**Comment by supsupsap** [comment]
thank you!
